contrib/docker/data/etc_init.d_artifactory_orig_2015-07-16_0912UTC.sh

```diff
@@ -0,0 +1,287 @@
+#!/bin/bash
```

Comment:
this file is just the "original" artifactory startup file at that point in time. It's not used by the puppet module at all, it's here as a reference for what I changed.
I am also hoping that JFrog will incorporate those changes into their "official" docker container, because I fear nobody has actually used the container in a deployment until now ...